### PR TITLE
Update 'how to contribute' instractions

### PR DESCRIPTION
**Pull request by ItayHarshuv**
changed the instractions for how to get started that you have to use yarn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,11 +24,14 @@ The easiest way to contribute to OpenEar is creating some new exercises on the e
...
 
 1. Fork the repository
-2. open the repository root
-3. Install ionic globally by running `npm i ionic -g`
-4. Run `yarn` or `npm i` (you may need to install node if you don't have it installed already)
-5. Run `yarn start`
-6. App should be served in the browser!
+2. Open the repository root
+3. Download and install node at https://nodejs.org/en/download/
+4. Open nodejs directory
```
Comment (ShacharHarshuv) [inline README.md:29]: Why is this needed?

File: README.md [hunk 2]
```diff
@@ -24,11 +24,14 @@ The easiest way to contribute to OpenEar is creating some new exercises on the e
 To contribute follow the following steps.
 
 1. Fork the repository
-2. open the repository root
-3. Install ionic globally by running `npm i ionic -g`
-4. Run `yarn` or `npm i` (you may need to install node if you don't have it installed already)
-5. Run `yarn start`
-6. App should be served in the browser!
+2. Open the repository root
+3. Download and install node at https://nodejs.org/en/download/
```
Comment (ShacharHarshuv) [inline README.md:28]:
```suggestion
3. Download and install node at https://nodejs.org/en/download/ (If not already installed)
```

File: README.md [hunk 3]
```diff
@@ -24,11 +24,14 @@ The easiest way to contribute to OpenEar is creating some new exercises on the e
...
+4. Open nodejs directory
+5. Install ionic globally by running `npm i ionic -g`
+6. Install yarn globally by running `npm install --global yarn`
```
Comment (ShacharHarshuv) [inline README.md:31]:
```suggestion
6. Install yarn globally by running `npm install --global yarn` (If not already installed)
```

**Review by ShacharHarshuv — COMMENTED** [review]
Note you have conflicts. Make sure to resolve them properly.

File: README.md (hunk 1 above)
Comment (ItayHarshuv) [reply README.md:29]: For me it didn't worked at first but when I changed directory to nodejs it did.

Comment (ShacharHarshuv) [reply README.md:29]: You mean "npm" was not defined as an [Environment Variable](https://en.wikipedia.org/wiki/Environment_variable) ?
That's something that should have worked after the installation, and is definitely not a required part of node installation was done properly. 

Anyway we can remove this step from the readme.

File: README.md [hunk 4]
```diff
@@ -24,11 +24,14 @@ The easiest way to contribute to OpenEar is creating some new exercises on the e
...
 1. Fork the repository
-2. open the repository root
-3. Install ionic globally by running `npm i @ionic/cli -g` (you may need to install node if you don't have it installed already)
-4. Run `yarn` or `npm i`
-5. Run `yarn start`
-6. App should be served in the browser!
+2. Open the repository root
+3. Download and install node at https://nodejs.org/en/download/ (If not already installed)
+4. Open nodejs directory
+5. Install ionic globally by running `npm i ionic -g`
```
Comment (ShacharHarshuv) [inline README.md:30]: Seems like you resolved conflicts incorrectly because you overwrote this change: https://github.com/ShacharHarshuv/open-ear/commit/cf4ec538762e2b41b1851f238aac39d1573e209c

**Review by ShacharHarshuv — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ItayHarshuv) [reply README.md:29]: Fixed

File: README.md (hunk 4 above)
Comment (ItayHarshuv) [reply README.md:30]: Fixed
